x-pack/examples/ui_actions_enhanced_examples/public/drilldowns/dashboard_hello_world_drilldown/index.tsx

```diff
@@ -37,11 +36,11 @@ export class DashboardHelloWorldDrilldown implements Drilldown<Config, ActionCon
     return [VALUE_CLICK_TRIGGER, SELECT_RANGE_TRIGGER];
   }
 
-  private readonly ReactCollectConfig: React.FC<CollectConfigProps<Config>> = ({
+  public readonly CollectConfig = ({
     config,
     onConfig,
     context,
-  }) => (
+  }: CollectConfigProps<Config, any>) => (
```

Comment:
```suggestion
  }: CollectConfigProps<Config>) => (
```

this seems to be working fine